api/src/pcapi/routes/native/v1/serialization/favorites.py

```diff
@@ -50,6 +51,7 @@ class Config:
     @classmethod
     def from_orm(cls, offer: Offer) -> "FavoriteOfferResponse":
         offer.coordinates = {"latitude": offer.venue.latitude, "longitude": offer.venue.longitude}
+        offer.venueName = offer.venue.publicName or offer.venue.name
```

Comment:
On a depuis quelques temps une propriété qui permet d'éviter cette expression `or` dont notre code était truffé : 

```suggestion
        offer.venueName = offer.venue.common_name
```